pkg/sql/catalog/tabledesc/validate.go

```diff
@@ -596,13 +596,13 @@ func (desc *wrapper) ValidateSelf(vea catalog.ValidationErrorAccumulator) {
 		// m we are considering right now is invalid.
 		if foundAlterPK {
 			if alterPKMutation == m.MutationID {
-				vea.Report(unimplemented.NewWithIssue(
-					45615,
+				// Tracked with issue 45615.
+				vea.Report(pgerror.Newf(pgcode.FeatureNotSupported,
```

Comment:
unimplemented.NewWithIssue is probably better. removing the issue number removes telemetry and the hint that points to the issue.

you could use unimplemented.New for the other stuff